doc/examples/external_simulators.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Script based external simulators"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we demonstrate the usage of the script based external simulators, summary statistics, and distance functions features.\n",
+    "\n",
+    "These allow to use near-arbitrary programing languages and output for pyABC. The main concept is that all communication happens via the file system. This comes at the cost of a considerable overhead, making this feature not optimal for models with a low simulation time. For more expensive models, the overhead should be negligible."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebooke replicates the using_R notebook, but circumvents usage of the rpy2 package. Note the considerably longer runtime though."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/yannik/anaconda3/lib/python3.7/site-packages/distributed/config.py:20: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
+      "  defaults = yaml.load(f)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pyabc\n",
+    "import pyabc.external"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we define model, summary statistics and distance. Note that if possible, alternatively FileIdSumStat can be used to read in the summary statistics directly to python and then use a python based distance function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = pyabc.external.ExternalModel(executable=\"Rscript\", file=\"rmodel/model.r\")\n",
+    "sumstat = pyabc.external.ExternalSumStat(executable=\"Rscript\", file=\"rmodel/sumstat.r\")\n",
+    "distance = pyabc.external.ExternalDistance(executable=\"Rscript\", file=\"rmodel/distance.r\")\n",
```

Comment:
can you add here a little comment indicating that the experimental data is already included in the definition of "distance.r" ? Otherwise the user needs to check the using_R script (which I did, because I did not know where the data was coming from in the last plot). May be good to shortly indicate this.